crypto/asymmetric_keys/Makefile

```diff
@@ -67,6 +67,8 @@ obj-$(CONFIG_PKCS7_TEST_KEY) += pkcs7_test_key.o
 pkcs7_test_key-y := \
 	pkcs7_key_type.o
 
+obj-y += tests/
+
```

Comment:
Also note that the patch description mentions `lib/tests`, but should actually mention `crypto/asymmetric_keys/tests`.
Keep in mind everything below `---` will be removed from the git log.